velox/common/memory/SharedArbitrator.cpp

```diff
@@ -466,6 +483,19 @@ bool SharedArbitrator::handleOOM(
   return true;
 }
 
+void SharedArbitrator::grow(
+    MemoryPool* pool,
+    uint64_t growBytes,
+    uint64_t reservationBytes) {
+  const auto ret = pool->grow(growBytes, reservationBytes);
+  VELOX_CHECK(
+      ret,
+      "Failed to grow pool {} with {} and commit {} used reservation",
+      pool->name(),
+      succinctBytes(growBytes),
+      succinctBytes(reservationBytes));
```

Comment:
`VELOX_CHECK(pool->grow(), ...)`

Also shall we rename it to checkedGrow to differentiate? The invocation in growCapacity can just not call this one but pool.grow directly, instead

Reply:
It is not good to put a production function call inside check macro. Yeah, rename to checkedGrow thanks!